File: example/src/main/java/com/github/simonpercic/oklogexample/data/api/model/request/show/CreateShowRequest.java
package com.github.simonpercic.oklogexample.data.api.model.request.show;

/**
 * @author Simon Percic <a href="https://github.com/simonpercic">https://github.com/simonpercic</a>
 * @noinspection FieldCanBeLocal
 */
public class CreateShowRequest {

    private final String name;

    private final int runtime;

    private final ShowNetworkRequest network;

    public CreateShowRequest(String name, int runtime, ShowNetworkRequest network) {
        this.name = name;
        this.runtime = runtime;
        this.network = network;
    }
}

File: example/src/main/kotlin/com/github/simonpercic/oklogexample/data/api/model/request/show/CreateShowRequest.kt
package com.github.simonpercic.oklogexample.data.api.model.request.show

/**
 * @author Simon Percic [https://github.com/simonpercic](https://github.com/simonpercic)
 */
data class CreateShowRequest(private val name: String, private val runtime: Int, private val network: ShowNetworkRequest)
